api/router.go

```diff
@@ -52,14 +52,14 @@ var routes = map[string]map[string]handler{
 		"/images/{name:.*}/push":        proxyImage,
 		"/images/{name:.*}/tag":         proxyImageAndForceRefresh,
 		"/containers/create":            postContainersCreate,
-		"/containers/{name:.*}/kill":    proxyContainer,
-		"/containers/{name:.*}/pause":   proxyContainer,
-		"/containers/{name:.*}/unpause": proxyContainer,
+		"/containers/{name:.*}/kill":    proxyContainerAndForceRefresh,
```

Comment:
@vieux Could you look at the routes and make sure we didn't miss anything?

For instance, I'm wondering if `/containers/{name:.*}/export` needs a refresh.

Also, what about `/exec/{execid:.*}/start`?